main.cpp

```diff
@@ -1,23 +1,197 @@
 #include <iostream>
 #include <string>
 #include <string_view>
+#include <sstream>
+#include <vector>
 #include <algorithm>
 
 namespace
 {
+int random_no(unsigned int range)
+{
+	srand(time(NULL));
+	int secret_num = rand() % range;
+	return secret_num;
+}
+
 std::string to_lowercase(std::string_view word)
 {
 	std::string lowercased = word.begin();
 	std::transform(word.begin(), word.end(), lowercased.begin(), tolower);
 	return lowercased;
 }
+
+void initialize_combolist(std::stringstream& combo_names, std::vector<std::string>& combo_list)
+{
+	std::string combo_name;
+	while (getline(combo_names, combo_name, ' '))
+	{
+		combo_list.push_back(combo_name);
+	}
+}
+
+void display_rules()
+{
+	std::cout << "> RULES: \n";
+	std::cout << "> 1. If a number is guessed number of tries goes down by 1\n";
+	std::cout << "> 2. If a character that is guessed is not in the word \n";
+	std::cout << "     the number of tries goes down by 1\n\n";
+}
+
+void start_screen()
+{
+	std::cout << "\n== HANGMAN ==\n";
+	std::cout << "> Press 0 to quit at any time\n\n";
+	std::cout << "> OBJECTIVE: \n";
+	std::cout << "> Guess the word before running out of tries\n\n";
+	display_rules();
+}
+
+void ask_for_input(char* dest)
+{
+	std::cout << "> ";
+	std::cin >> *dest;
+	std::cout << "\n";
+}
+
+void update_letters_used(std::vector<int>& score, int pos)
+{
+	score[pos]++;
+}
+
+void fill_letter(std::string& word, std::string& hangman, int pos)
+{
+	hangman[pos] = word[pos];
+}
+
+// used to display the next combo name
+void display_combo_name(std::string combo_name)
+{
+	std::cout << combo_name << '\n';
+}
+
+void display_next_combo_name(int tries, std::vector<std::string>& combolist, int combolist_length)
+{
+	if (tries < combolist_length)
+	{
+		display_combo_name(combolist[tries]);
+	}
+	else
+	{
+		display_combo_name(combolist[combolist_length - 1]);
+	}
+}
 } // namespace
 
 int main()
 {
-	std::string_view const test = "Heyo\n";
-	std::string lowercased_test = to_lowercase(test);
+	char usr_input = ' ';
+
+	// # of failed attempts
+	int f_tries = 0;
```

Comment:
Single letter prefixes have special interpretations, eg in Hungarian notation. Avoid them for such names.